README.md

```diff
@@ -53,7 +53,7 @@ The T-Route routing module converts water flux data (e.g., overland flow, later
 
 ### Coastal models
 
-To produce total water level forecasts, OWP is adapting two coastal models to work with NextGen: the  [Semi-implicit Cross-scale Hydroscience Integrated System Model (SCHISM)](https://schism-dev.github.io/schism/master/index.html) and [D-Flow Flexible Mesh (FM)](https://www.deltares.nl/en/software-and-data/products/delft3d-fm-suite/modules/d-flow-flexible-mesh). The initial development stages for the coastal model BMIs have been completed and their respective GitHub repositories are stored in the following pathways: SCHISM - [https://github.com/LynkerIntel/SCHISM_BMI](https://github.com/LynkerIntel/SCHISM_BMI)  ; DFlowFM - [https://github.com/LynkerIntel/DFlowFM_BMI](https://github.com/LynkerIntel/DFlowFM_BMI) . Once the NextGen framework evaluation is complete with the SCHISM Total Water Level (TWL) capability, the SCHISM BMI repository will be merged with the SCHISM master repository (https://github.com/schism-dev/schism) in the near future. For the DFlowFM model, it's BMI development is an entire seperate branch of the BMI code that is incompatible with the Deltares DFlowFM master branch and that will remain the case until the DFlowFM EC-module is redeveloped entirely for BMI compatibility. For now, the NOAA DFlowFM BMI development is a frozen DFlowFM version for research and development initiatives under the NextGen project in the future with the TWL capability. 
+To produce total water level forecasts, OWP is adapting two coastal models to work with NextGen: the  [Semi-implicit Cross-scale Hydroscience Integrated System Model (SCHISM)](https://schism-dev.github.io/schism/master/index.html) and [D-Flow Flexible Mesh (FM)](https://www.deltares.nl/en/software-and-data/products/delft3d-fm-suite/modules/d-flow-flexible-mesh). The initial development stages for the coastal model BMIs have been completed and their respective GitHub repositories are stored in the following pathways: SCHISM - [https://github.com/LynkerIntel/SCHISM_BMI](https://github.com/LynkerIntel/SCHISM_BMI)  ; DFlowFM - [https://github.com/LynkerIntel/DFlowFM_BMI](https://github.com/LynkerIntel/DFlowFM_BMI) . Once the NextGen framework evaluation is complete with the SCHISM Total Water Level (TWL) capability, the SCHISM BMI repository will be merged with the SCHISM master repository (https://github.com/schism-dev/schism) in the near future. For the DFlowFM model, its BMI development is an entire separate branch of the BMI code that is incompatible with the Deltares DFlowFM master branch and that will remain the case until the DFlowFM EC-module is redeveloped entirely for BMI compatibility. For now, the NOAA DFlowFM BMI development is a frozen DFlowFM version for research and development initiatives under the NextGen project in the future with the TWL capability.
```

Comment:
I'm probably missing something obvious, but I can't figure out the difference(s) between the new line and the old.

There are some outstanding formatting issues, too:
- Double spaces (e.g., `the  [Semi`)
- space . space (e.g., `BMI) . Once`)

And the last sentence does not make grammatical sense.

Reply:
Looking at `git show --word-diff`, the changes were to fix `it's` and `seperate`. I'll fix the spaces too.

@jduckerOWP will have to clarify exactly what meaning was intended for the final sentence. Possibly, some words or phrase just accidentally got cut.

Reply:
Ooh, the Github rendered view of the PR actually does indicate the word-level diff rather than highlighting the whole line. Top-right corner of the file display, the little paper icon next to the `<>` icon, to the left of the 'Viewed' checkbox.

Reply:
Thanks for the pro-tips, @PhilMiller! They're both more useful than me squinting at the green blob.

Reply:
@PhilMiller thank you for fixing the spacing with this paragraph block about the coastal model BMI development. Apologies to begin with some of the typos and grammatical issues as I had to scramble to get this documentation in before the end of the contract. We can reword the last sentence as the following: `The DFlowFM BMI source code in the GitHub repository will remain as a separate [branch](https://git.deltares.nl/oss/delft3d/-/merge_requests/493) of the Deltares GitLab repository that was modified strictly for BMI development within the NextGen framework.` How does that statement sound @PhilMiller @SnowHydrology ?